Makefile

```diff
@@ -52,7 +52,7 @@ asm: zsbl.asm fsbl.asm
 
 lib/version.c: .git/HEAD .git/index
 	echo "const char *gitid = \"$(shell git describe --always --dirty)\";" > lib/version.c
-	echo "const char *gitdate = \"$(shell git log -n 1 --date=short --format=format:"%ad.%h" HEAD)\";" >> lib/version.c
+	echo "const char *gitdate = \"$(shell git log -n 1 --date=short --format=format:"%ad.%h" --no-show-signatures HEAD)\";" >> lib/version.c
```

Comment:
Looks like git version in OE doesn't support `--no-show-signatures`. That seems to be available starting "2.10.0" version. Should we bump git version in OE?

Reply:
This failed in Fedora/RISCV after git updated to 2.22.0 (worked until 2.21.0). There is a typo in the option name, it should be `--no-show-signature` (no `s` at the end).

See: https://github.com/git/git/blob/master/revision.c#L2209